com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ExtenderUtil.java

```diff
@@ -504,6 +527,8 @@ private static IResource getPropertyResource(Project project, BobProjectProperti
             sources.add( new FSExtenderResource( resource ) );
             sources.addAll( listFilesRecursive( project, extension + "/include/" ) );
             sources.addAll( listFilesRecursive( project, extension + "/src/") );
+            sources.addAll( listFilesRecursive( project, extension + "/commonsrc/") );
```

Comment:
Shared between engine + plugins

The setup is required since windows dll's need special build flags, that make them incompatible with static libraries. (/MD vs /MT)
(At least that's how I remember it)